server/methods/saveUserPreferences.js

```diff
@@ -50,11 +50,7 @@ Meteor.methods({
 			} else {
 				delete preferences.mergeChannels;
 			}
-			if (settings.unreadRoomsMode) {
```

Comment:
Well I checked it personally. This change was not made by me. You can revert the changes by default for this part.